readme.md

```diff
@@ -35,14 +52,18 @@ Type: `string`
 
 String with CSS.
 
+
 ## options
 
-### all
+### preserve
 
-Type: `boolean`  
-Default: `false`
+Type: `boolean`, `RegExp`, or `function` 
+Default: `true`
 
-Whether *important* CSS comments *(those starting with `/*!`)* should be stripped.
+- `preserve: true` &mdash; (default) preserve comments that use the `/*! */` syntax;
+- `preserve: false` &mdash; strip all comments;
+- `preserve: [RegExp]` &mdash; preserve comments that match a regular expression. The comment text but not the comment syntax (`/**/`) will be tested by the RegExp.
+- `preserve: function (comment) { ... }` &mdash; a function that returns `true` to preserve the comment or `false` to strip it. The comment is invoked with a single argument, the string found between the comment syntax, `/**/`.
 
 
```

Comment:
@Qix- Why? It's unrelated to this PR and 2 lines between sections of the readme is consistent. That's the way we do it everywhere.